user_guide_src/source/changelogs/v4.5.0.rst

```diff
@@ -18,6 +18,8 @@ Highlights
 BREAKING
 ********
 
+- **AutoRouting Legacy:** Add a condition to check if the controller exists.
+
```

Comment:
The place is not correct. 

```diff
--- a/user_guide_src/source/changelogs/v4.5.0.rst
+++ b/user_guide_src/source/changelogs/v4.5.0.rst
@@ -18,8 +18,6 @@ Highlights
 BREAKING
 ********
 
-- **AutoRouting Legacy:** Add a condition to check if the controller exists.
-
 Behavior Changes
 ================
 
@@ -78,6 +76,8 @@ Return Type Changes
 Others
 ------
 
+- **AutoRouting Legacy:** Changed so that a ``PageNotFoundException`` is thrown
+  if the controller corresponding to the request URI does not exist.
 - **Logger:** The method signatures of the methods in ``CodeIgniter\Log\Logger``
   that implements the PSR-3 interface have been fixed. The ``bool`` return
   types are changed to ``void``. The ``$message`` parameters now have
```

Reply:
Done.